refarch-integrations/refarch-cosys-integration/refarch-cosys-integration-example/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,35 @@
+refarch:
+  s3:
+    client:
+      document-storage-url: http://localhost:8086
+      enable-security: true
+  cosys:
+    url: https://cosysk007.srv.muenchen.de/cosys-eai/rest
```

Comment:
Can we somehow get rid of that internal reference?

Reply:
Done